app.js

```diff
@@ -0,0 +1,43 @@
+"use strict";
+
+(function(){
+  angular.module('toDoApp', []).controller('list_controller', ListController);
```

Comment:
I would suggest putting these `.module` and `.controller` on separate lines for readability!